.github/workflows/check-lambda-js.yml

```diff
@@ -10,15 +10,16 @@ on:
       - main
       - development
```

Comment:
The UI _does_ disable that option, but it seems to send to the backend admin-related permissions (even if they do not change the status quo), which the backend rejects. So, at the moment, I do not seem to be able to change any user permissions of any kind as a Board ED.

Reply:
Ah, I think then **1)** the API docs need to be updated, and / or **2)** the backend could be updated to be idempotent, and / or **3)** the frontend can conditionally exclude request props.

Let me know if that tracks with your thinking.

If item 3 is urgent, it can be a new ticket. Otherwise it will be related to upcoming #325 (currently top of the backlog) and a note can be added to that ticket.

Reply:
Let's add to that ticket. @jusdino informed me that this was an intentional API decision, so we should probably do #1 and #3 on your list.

Reply:
Gotcha. I can't actually edit #325. Do you mind adding the following to the original ticket notes:

- Update API docs to reflect that the top-level `actions` prop can only conditionally be included if user is compact ED
- Update frontend data layer to prevent top-level `actions` prop from being included in payload if logged-in user is not a compact ED

Reply:
You got it!